pkg/xray/resource_xray_settings.go

```diff
@@ -0,0 +1,99 @@
+package xray
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceXraySettings() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceXrayDbSyncTimeCreate,
+		ReadContext:   resourceXrayDbSyncTimeRead,
+		UpdateContext: resourceXrayDbSyncTimeUpdate,
+		DeleteContext: resourceXrayDbSyncTimeDelete,
+		Description:   "Provides an Xray DB Sync Time resource.",
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"db_sync_updates_time": {
+				Type:             schema.TypeString,
+				Required:         true,
+				Description:      "The time of the Xray DB sync daily update job.",
```

Comment:
In the description, give the exact format

Reply:
Fixed